torch/_functorch/config.py

```diff
@@ -88,6 +88,13 @@
 # a fusion can be expensive.
 ban_recompute_reductions = True
 
+memory_budget = 1.0
```

Comment:
Do you have any sense on what the compile time/ perf tradeoffs are for dp vs ilp ?

Reply:
Ilp strictly dominates DP in both quality + runtime. I just don't want to add an unnecessary scipy dependency.